src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+const expenseService = require('../services/expenses.service');
+const userService = require('../services/user.service');
+
+function filterRequestBody(body, allowedFields) {
+  const result = {};
+
+  for (const field of allowedFields) {
+    if (body[field]) {
+      result[field] = body[field];
+    }
+  }
+
+  return result;
+}
+
+const getAllExpenses = (req, res) => {
+  const expenses = expenseService.getAllExpenses(req.query);
+
+  res.send(expenses);
+};
+
+const getExpenseById = (req, res) => {
+  const { expenseId } = req.params;
+  const foundExpense = expenseService.getExpenseById(expenseId);
+
+  if (!foundExpense) {
+    res.sendStatus(404);
+
+    return;
```

Comment:
In addition to setting the status, you should also add information for the user.
```suggestion
      return res.status(404).send("No expenses found.");
```